CHANGES.rst

```diff
@@ -118,6 +118,14 @@ srctype
 
 - Add command line option to override source type [#6720]
 
+1.4.6 (2022-03-25)
```

Comment:
Is it too late to add (cherry pick) this change into the existing 1.4.5 tag/branch, rather than having to create yet another new one?